readthedocs/gold/forms.py

```diff
@@ -88,6 +89,18 @@ def __init__(self, *args, **kwargs):
         self.projects = kwargs.pop('projects', None)
         super(GoldProjectForm, self).__init__(*args, **kwargs)
 
+    def clean_project(self):
+        cleaned_data = super(GoldProjectForm, self).clean()
+        project_slug = cleaned_data.get('project', '')
+        project_instance = Project.objects.filter(slug=project_slug)
+
+        # Checking if the project with the entered slug
+        # is absent or present in the database.
```

Comment:
We don't need this comment, is clear what we are checking

Reply:
Working on it.